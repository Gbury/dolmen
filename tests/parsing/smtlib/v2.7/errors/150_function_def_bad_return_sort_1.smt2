(set-logic ALL)
(define-fun-rec foo () _)
