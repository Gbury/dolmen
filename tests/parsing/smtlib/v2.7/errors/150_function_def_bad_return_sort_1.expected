File "tests/parsing/smtlib/v2.7/errors/150_function_def_bad_return_sort_1.smt2", line 2, character 23-24:
2 | (define-fun-rec foo () _)
                           ^
Error while parsing a function definition, read the reserved word '_',
      but expected a sort for the return type of the function.
